python/lsst/db/utils.py

```diff
@@ -372,3 +373,17 @@ def userExists(conn, userName, hostName):
                 (userName, hostName)).scalar() == 1
     else:
         raise NoSuchModuleError(conn.engine.url.get_backend_name())
+
+
+#### Unclassified functions ########################################################
+def typeCode2Name(conn, code):
```

Comment:
This has portability issues, I believe `code` above is not a portable sqlalchemy thing. As I mentioned in qserv branch this is closely related to what Brian wants to do for dax_dbserv, maybe we should merge this together with Brian's development.